### ISSUE TITLE
Can you please update documentation to say "run nvm use x.x.x in administrator mode"

### ISSUE THREAD

**Issue by brianlmerritt** [coreybutler/nvm-windows#566]
Windows 10.0.18363 clean install of NVM

After a clean install (with no node already installed) I ran `nvm install 12.18.3` and then `nvm use 12.18.3` in git bash terminal.

I got an error message C:\ etc etc \cmd no program exists for this request.  If I ignore the error and run npm I get a command not found error.

As soon as I run cmd as an administrator the nvm use works fine.

Great program (bonus points for implementing nvm ls-remote) but this might help people who can't figure it out themselves.

**Comment by coreybutler** [comment]
[This note](https://github.com/coreybutler/nvm-windows#usage) already mentions that. 

Do you have a suggestion for making this more apparent? I'd be happy to add something somewhere else if it will help save people some time/frustration.

**Comment by devna13** [comment]
I'm having same problem and using PS in admin mode doesn't resolve the issue. (Windows 10.0.18363)

![image](https://user-images.githubusercontent.com/4080343/92622186-a0be1d80-f279-11ea-8e8c-4ee839b970b5.png)

**Comment by bmcgoncalves** [comment]
I'm new to nodejs and in the last two days i've started to install all my dev environment.

I've installed nvm for windows with admin rights using an administrator user. When I use powershell with admin rights it's all ok.

The problem is when i use terminal in vscode. I don't want to start vscode with admin rights, so node it's not available in vscode.

There's some config or command to install node versions in another user? I have my user and my company gave me another user with admin rights to install software.

**Comment by coreybutler** [comment]
NVM for Windows works by using symlinks to point to the "active" Node installation. This is what makes it work when `node whatever.js` is executed. Unfortunately, the Windows operating system requires admin privileges to create symlinks.

There are only two alternatives to this approach. The first is to change the installation `PATH` variable. This is incredibly risky because any error can cause the variable to be set to an empty string.... meaning no programs will be found. For example,older versions of Windows have a limited number of characters that the `PATH` can contain. Trying to write more than the allotted number of characters causes this behavior. That is an unacceptable risk.

The other option is to shim node. This is how some of the other version managers work, and from what I can tell, it is the overwhelming source of operational problems. A shim does not necessarily pickup the permissions of the user from which it is called, so node itself may appear to behave in unexpected ways. There is simply too much churn.

Bottom line: you need admin access to do this.... just like you need admin access to actually install Node on a Windows system.

**Comment by coreybutler** [comment]
Closing, since I think this is already addressed. @brianlmerritt - still happy to take suggestions if you have them.

**Comment by brianlmerritt** [comment]
Hi, sorry have been away.

I hit the issue because I didn't read as far as usage.  I know how to run nvm install xxxx, just never used nvm on windows before.

I would change 

`tl;dr nvm, but for Windows, with an installer. Download Now! This has always been a node version manager, not an io.js manager, so there is no back-support for io.js. However, node 4+ is supported.` 

to

`tl;dr nvm, but for Windows, with an installer. Download Now! This has always been a node version manager, not an io.js manager, so there is no back-support for io.js. However, node 4+ is supported. Remember when running "nvm install" you must do this as a Windows admin.`

**Comment by coreybutler** [comment]
Thanks @brianlmerritt - that seems reasonable. I have updated the README. Thanks for the suggestion!
